lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -65,9 +66,10 @@
                         },
                         error: function () {
                             view.showNotificationMessage(
-                                view.indicators.error +
+                                HtmlUtils.joinHtml(
+                                view.indicators.error,
```

Comment:
Indent params 1, and put closing `)` on its own line at the same indent level as HtmlUtils.